CredentialProvider.Microsoft/Resources.resx

```diff
@@ -350,93 +350,99 @@ Windows Integrated Authentication
+    <value>Attempting to exchange the bearer token for an Azure DevOps session token.</value>
+  </data>
+  <data name="NotRunningBearerTokenProvider" xml:space="preserve">
+    <value>Not running bearer token provider '{0}'</value>
+  </data>
+  <data name="DeviceFlowTimedOut" xml:space="preserve">
+    <value>Device flow authentication failed. User was presented with device flow, but didn't react within {0} seconds.</value>
+  </data>
+  <data name="CouldNotParseDeviceFlowTimeoutOverride" xml:space="preserve">
+    <value>Could not parse Device Flow Timeout override: {0}</value>
+  </data>
+  <data name="BuildTaskCredProviderIsUsedError" xml:space="preserve">
+    <value>This credential provider must not run if any build task environment variables are set. Variables: VSS_NUGET_URI_PREFIXES, VSS_NUGET_ACCESSTOKEN, VSS_NUGET_EXTERNAL_FEED_ENDPOINTS</value>
+  </data>
+  <data name="BuildTaskEndpointNoMatchingUrl" xml:space="preserve">
+    <value>Environment variable VSS_NUGET_EXTERNAL_FEED_ENDPOINTS did not have credentials for endpoint {0}</value>
+  </data>
+  <data name="BuildTaskEndpointMatchingUrlFound" xml:space="preserve">
+    <value>Found credentials for endpoint {0}</value>
+  </data>
+  <data name="CancelMessage" xml:space="preserve">
+    <value>Canceling credential provider</value>
+  </data>
+  <data name="ResponseHandlerException" xml:space="preserve">
+    <value>Caught exception processing {0}  (RequestId: {1})</value>
+  </data>
+  <data name="CacheException" xml:space="preserve">
+    <value>Exception happened when trying to get cached token. Invalidating cache. Message: {0}</value>
+  </data>
+  <data name="BuildTaskNoMatchingPrefixes" xml:space="preserve">
+    <value>none</value>
+  </data>
+  <data name="RequestHandlerCancelingExceptionMessage" xml:space="preserve">
+    <value>Ignoring a cancellation exception during request handling. Exception: {0}, Message: {1}</value>
+  </data>
+  <data name="ShuttingDown" xml:space="preserve">
+    <value>Credential provider shutting down.</value>
+  </data>
+  <data name="StartRequestHandlerStopwatch" xml:space="preserve">
+    <value>Starting stopwatch for {1} {2}</value>
+  </data>
+  <data name="TimeElapsedAfterSendingResponse" xml:space="preserve">
+    <value>Time elapsed in milliseconds after sending response '{0}' '{1}': {2}</value>
+  </data>
+  <data name="SessionTokenCacheMutexFail" xml:space="preserve">
+    <value>Unable to gain a lock on the credential cache.</value>
+  </data>
+  <data name="SessionTokenCacheMutexMiss" xml:space="preserve">
+    <value>Another instance of CredentialProvider is accessing the credential cache, waiting for it to become available.</value>
+  </data>
+  <data name="LogStartsAt" xml:space="preserve">
+    <value>Log starts at {0}</value>
+  </data>
+  <data name="WhileShuttingDown" xml:space="preserve">
+    <value>While the plugin was shutting down:</value>
+  </data>

```

Comment:
Are LogStartsAt and WhileShuttingDown the only "actual" changes in this file? Not sure if this is just GitHub being weird or if the file has some whitespace diffs

Reply:
yeah, I'm guessing VS changed the line endings

if you add `?w=1` to the URL, GitHub will ignore whitespace in diffs

Reply:
Nice, thanks! I didn't know about that one.